core/templates/services/angular-services.index.ts

```diff
@@ -0,0 +1,692 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Angular services index file.
+ */
+
+/* eslint-disable max-len */
```

Comment:
Not allowed, these are going to be made into per-line checks.

As mentioned before please fix the linter directly to exclude import statements from needing to adhere to the max-len rule.

Reply:
Done!